src/main/java/com/example/enterprise/service/IncomeService.java

```diff
@@ -7,6 +7,11 @@
 
 import java.util.List;
 
+/**
+ * The IncomeService class provides necessary logic for operation methods related to Income
+ * such as save, list, and search. It implements all the methods from Income service interface.
+ *
+ */
```

Comment:
Added Javadoc to the IncomeService class as it is a good coding convention to follow. It provides the explanation to an outsider that what that class is trying to do. 
Here is a link to an article: https://www.tutorialspoint.com/java/java_documentation.htm